pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -126,9 +126,21 @@ type OSDInfo struct {
 }
 
 type OrchestrationStatus struct {
-	OSDs    []OSDInfo `json:"osds"`
-	Status  string    `json:"status"`
-	Message string    `json:"message"`
+	OSDs         []OSDInfo `json:"osds"`
+	Status       string    `json:"status"`
+	PvcBackedOSD bool      `json:"pvc-backed-osd"`
+	Message      string    `json:"message"`
+}
+
+type OSDObject struct {
+	name           string
```

Comment:
can you call this `nodeName`? Otherwise it's ambiguous what name this should be.

Reply:
For OSDs based on PVCs we will need to not use the actual hostname in order to allow for portability. How about setting the `nodeName` to the name of the StorageClassDeviceSet? Then if we set `topologyAware: true`, the zone and/or region will also be added to the CRUSH map above this.

Reply:
> can you call this `nodeName`? Otherwise it's ambiguous what name this should be.

I believe `name` here could either be the name of the `node`, that is, the hostname (in existing behavior) or the name of the PVC. So keeping it as `nodeName` might still be ambiguous. Thoughts?

Reply:
True, `nodeName` is too specific since it could also be the pvc name. How about `crushHostname`? Then if you could add a comment on the property that it's the hostname in some scenarios and the pvc name in other scenarios.

Reply:
`crushHostname` makes sense